examples/advanced_agents/sandbox_agent/src/nat_sandbox_agent/__init__.py

```
@@ -0,0 +1,41 @@
+Usage:
+    # CLI mode
+    nat run configs/config.yaml
+
+    # Evaluation mode
+    nat eval configs/config_gaia.yaml
+"""
+
+__version__ = "0.1.0"
+
+# Import registration to enable NAT component discovery
+from nat_sandbox_agent import register  # noqa: F401
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)